### ISSUE TITLE
regexp detected as doublequoted string

### ISSUE THREAD

**Issue by eqv** [rubocop/rubocop#232]
a regexp begining and ending with " is reported as a string that should use single quotes instead.

``` ruby
# encoding: utf-8
a = /"([^"\\]|\\.)*"/
```

```
rubocop test.rb 
== test.rb ==
C:  2: Prefer single-quoted strings when you don't need string interpolation or special symbols.

1 file inspected, 1 offence detected
```

This is most likely due to the fact that the regex is parsed as:

``` ruby
1.9.3-p194 :006 > p Parser::CurrentRuby.parse('/"([^"\\]|\\.)*"/')
(regexp
  (str "\"([^\"\\]|\\.)*\"")
  (regopt))
```
